src/core/override-configuration.js

```diff
@@ -4,14 +4,12 @@
 // tweaks to a document before generating the snapshot, without mucking with the source.
 // For example, you can change the status and date by appending:
 //      ?specStatus=LC&publishDate=2012-03-15
-// Note that fields are separated by semicolons and not ampersands.
-// TODO
-//  There could probably be a UI for this to make it even simpler.
 import { pub, sub } from "./pubsubhub";
 
 export const name = "core/override-configuration";
 
 function overrideConfig(config) {
+  // For legacy reasons, we still support both ";" and "&"
```

Comment:
Do those reasons still exist?

Reply:
yes 😭 There are a ton of these little files in various repos:
https://github.com/w3c/payment-request/blob/gh-pages/echidna-manifest.txt#L1

notice the ";" there. They are passed to ReSpec.